From 96cf3dede02cbf58134115603209d863fc82a06c Mon Sep 17 00:00:00 2001
From: orbea <orbea@riseup.net>
Date: Wed, 13 Mar 2024 10:01:49 -0700
Subject: [PATCH] tpm_crypto: add missing openssl includes

This fixes the build with LibreSSL 3.9.0 where many implicit
declarations for BN_, EVP_ and RSA_ functions occur which were
implicitly included before.

Signed-off-by: orbea <orbea@riseup.net>
---
 src/tpm12/tpm_crypto.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/tpm12/tpm_crypto.c b/src/tpm12/tpm_crypto.c
index bcbaa7fc2..628e27cc2 100644
--- a/src/tpm12/tpm_crypto.c
+++ b/src/tpm12/tpm_crypto.c
@@ -48,6 +48,8 @@
 #include <openssl/rand.h>
 #include <openssl/sha.h>
 #include <openssl/engine.h>
+#include <openssl/evp.h>
+#include <openssl/rsa.h>
 
 #include "tpm_cryptoh.h"
 #include "tpm_debug.h"
